terraform/spec/fixtures/projects/nested_modules/main.tf

```diff
@@ -0,0 +1,5 @@
+module "github_terraform_read_only" {
+  source  = "terraform-aws-modules/iam/aws//modules/iam-assumable-role"
+  version = "4.0.0"
+  role_name        = "github-terraform-read-only"
```

Comment:
nit: I think this is non-essential. Is it possible to remove it?

Reply:
Yup removing!